include/emp/Evolve/Systematics.hpp

```diff
@@ -812,107 +1008,114 @@ namespace emp {
     int GetPhylogeneticDiversity() const {
       // As shown on page 5 of Faith 1992, when all branch lengths are equal the phylogenetic
       // diversity is the number of internal nodes plus the number of extant taxa - 1.
+      //int phylodiversity = ancestor_taxa.size() + active_taxa.size() -1;
+
       return ancestor_taxa.size() + active_taxa.size() - 1;
     }
 
-    /** This is a metric of how distinct @param tax is from the rest of the population.
+
+    /// @returns phylogenetic diversity if used without any arguments .
+    /// If you want to receive normalized data, you need to include the number of generations
+    /// your tree has (multiples of 10 from 10 to 100 are allowed)
```

Comment:
The normalization is still very preliminary/experimental (really need to circle back to that paper eventually). This was part of a WAVES project and we haven't implemented anything more nuanced yet. It could absolutely be set up to work more generally

Reply:
Gotcha. Added a comment below on the implementation. Supersedes this comment thread.

Reply:
Ah, based on below it might be worth clarifying this docstring then